public/app/features/alerting/AlertRuleList.tsx

```diff
@@ -58,9 +59,9 @@ export class AlertRuleList extends PureComponent<Props, any> {
     return 'all';
   }
 
-  onStateFilterChanged = (evt: React.ChangeEvent<HTMLSelectElement>) => {
+  onStateFilterChanged = (value: SelectableValue) => {
```

Comment:
Nit: maybe rename to option?
```suggestion
  onStateFilterChanged = (option: SelectableValue) => {
```